<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>account</class>
 <widget class="QWidget" name="account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>account</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2E2E2E;
    color: #FFFFFF;
    font-family: Roboto, sans-serif;
}

QLabel {
    font-size: 18px;
    color: #FF5733;
    padding: 10px;
    text-align: center;
}

QLineEdit {
    border: 2px solid #444;
    border-radius: 5px;
    padding: 8px;
    background-color: #1C1C1C;
    color: #FFFFFF;
    margin: 10px 0;
}

QPushButton {
    border: 2px solid #444;
    border-radius: 5px;
    background-color: #333;
    color: #FFFFFF;
    padding: 10px;
    margin: 5px 0;
}

QPushButton:hover {
    background-color: #FF5733;
}

QPushButton:pressed {
    background-color: #C70039;
}

QGroupBox {
    border: 2px solid #FF5733;
    border-radius: 10px;
    padding: 20px;
    margin: 20px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Вернуться назад</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ВСЕ МОИ ПРЕДМЕТЫ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Введите название предмета для удаления</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>УДАЛИТЬ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
